src/utils/select-styles.ts

```diff
@@ -57,6 +57,7 @@ const filterStyles = (theme: DefaultTheme): Props['styles'] => ({
   }),
   input: () => ({
     color: theme.colors.white,
+    display: 'none'
```

Comment:
How does this resolve the issue of placeholder being incorrectly positioned?

Reply:
fixed